app/controllers/items_controller.rb

```diff
@@ -78,14 +78,22 @@ def restore
 
   private
 
-  def clean_purchase_amount
+  def clean_item_value_in_cents
     return nil unless params[:item][:value_in_cents]
 
     params[:item][:value_in_cents] = params[:item][:value_in_cents].gsub(/[$,.]/, "")
   end
 
+  def clean_item_value_in_dollars
+    return nil unless params[:item][:value_in_dollars]
+
+    params[:item][:value_in_cents] = params[:item][:value_in_dollars].gsub(/[$,]/, "").to_d * 100
+    params[:item].delete(:value_in_dollars)
+  end
+
   def item_params
-    clean_purchase_amount
+    clean_item_value_in_cents
+    clean_item_value_in_dollars
```

Comment:
Not a blocker since it was already like this, but I'm not the biggest fan of rewriting params like this. Will refactor later.

Reply:
Do you have another idea to solve something similar without rewriting params? Curious to know.

Other ways that I can think of is:

1. javascript on the view layer: trickier and more difficult to test
2. migrating the DB column: need to migrate existing entries, and some other issues mentioned on another PR with this approach
3. additional DB column shadowing the cents value: It looks like we will still end up on rewriting params.

Reply:
Yeah, I don't see a clear way of doing it better as-is. Anyhow, you did the right thing by following the existing pattern :)